specs/packages/sdk.md

```diff
@@ -44,3 +44,36 @@ Sdk struct wraps logic from the other modules, creating public API using `wasm_b
 
 At the moment, the client of the Sdk is responsible for keeping storage in sync with the Sdk's wallet instance. What does that mean? The Namada extension has its own key storage, which is modified by user actions. Every time the extension storage changes (for example, when a new address is derived), we must manually sync the extension and Sdk wallet storage
 using functions provided by `wallet` module. This is something that will most likely change in the future.
+
+## How can you import and use the SDK?
+First you need to build the shared package. We do that using `wasm-pack`.
```

Comment:
I would maybe update this as follows (very minor differences):

First, we need to compile the [shared](https://github.com/anoma/namada-interface/tree/main/packages/shared/lib) package to a _wasm_ target, which is done using [wasm-pack](https://rustwasm.github.io/wasm-pack/).